### PR TITLE
Jtj 60 hot fix

### PR DESCRIPTION
**Pull request by tgi-wtag**
In this PR i had to do e quick fix in the en.yml file (my code in master wasn't runnable die to wrongfuly fixed merging problems. 

Other than that i added forms styling, given that this application consists of many forms it looks like a loot of changed code, but really its nearly the same for all forms.

[JTJ-60.zip](https://github.com/tgi-wtag/Journey-tg/files/13367894/JTJ-60.zip)

Thank you for reviewing my code ! 🌻

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -35,20 +35,21 @@
...
+            <%= check_box_tag("task_status", value = "1", checked = (task.status == 'finished'), disabled: true) %>
+          </div>
+          <div>
+            <%= task.title %>
+          </div>
+        </li>
+      <% end %>
+    </li>
+  <% end %>
+</div>
```
Comment (evk-wtag) [inline app/views/pages/home.html.erb:55]: New line please

File: app/views/tasks/show.html.erb [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+    <p class="show-label"><strong><%= t('tasks.status') %>:</strong></p>
+    <p class="show-value"><%= @task.status %></p>
+
+    <div class="buttons-container">
+      <%= link_to t('tasks.back_to_tasks'), user_tasks_path(@user), class: 'button' %>
+      <%= link_to t('tasks.edit'), edit_user_task_path(@user, @task), class: 'button' %>
+      <%= link_to t('tasks.delete'), user_task_path(@user, @task), data: { turbo_method: :delete }, class: 'button' %>
+    </div>
+  </div>
+
```
Comment (evk-wtag) [inline app/views/tasks/show.html.erb:19]: Ditto

**Review by evk-wtag — APPROVED** [review]

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (tgi-wtag) [reply app/views/pages/home.html.erb]: thank you

File: app/views/tasks/show.html.erb (hunk 2 above)
Comment (tgi-wtag) [reply app/views/tasks/show.html.erb]: thank you

**Review by lra-wtag — APPROVED** [review]

**Pull request merged**